src/LazyRender.jsx

```diff
@@ -111,6 +114,19 @@ var LazyRender = React.createClass({
     });
   },
 
+  componentDidUpdate: function() {
+    //important to update the child height in the case that the children change(example: ajax call for data)
+    if(this.state.childHeight !== this.getChildHeight())
+      this.setState({childHeight: this.getChildHeight()});
```

Comment:
Won't this cause cascading renders since this will cause `componentDidUpdate` to be called again?